back/db/migrate/20230517145937_update_que_tables_to_version5.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+class UpdateQueTablesToVersion5 < ActiveRecord::Migration[6.1]
+  def up
+    if Rails.env.development? || Rails.env.test?
+      # We need to run a slightly different migration in development and test
+      # environments because some objects that `que` expects to be present are very
+      # likely to be missing. This is because `schema.rb` cannot represent certain
+      # database constructs, such as triggers and functions. Therefore, a simple
+      # `db:reset` removes those objects.
+      Que.execute(DEV_QUE_SQL_MIGRATION_5)
```

Comment:
Does Que function without these? Asking because `db:migrate` is not typically run in the dev setup, so we should expect these not to be there at all.

Reply:
I guess it does. Otherwise, we would have noticed earlier, no? I can double check.